### ISSUE TITLE
Keyboard accessibility - Focus order when tabbing 

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#162]
Related issue #101 
When tabbing the focus order should be:
1st Header elements
2nd side navigation menu elements
3rd. middle column elements
4th right column element
5th Footer elements

This worked in the original
